src/layers/VivViewerLayer/utils.js

```diff
@@ -39,12 +41,18 @@ export function renderSubLayers(props) {
       top
     ],
     channelData: data,
+    pickable,
+    // Uncomment to help debugging - shades the tile being hovered over.
+    // autoHighlight: true,
+    // highlightColor: [80, 80, 80, 50],
+    data: null,
```

Comment:
What is this prop?

Reply:
`data` has a very specific purpose in deck.gl (as a way of passing in vertex attributes or information that leads to vertex attributes) so setting it to null just makes it clear that we have none (or that they are inferred within the layer).